packages/govuk-frontend/src/govuk/components/character-count/character-count.jsdom.test.mjs

```diff
@@ -3,17 +3,36 @@ import { CharacterCount } from './character-count.mjs'
 describe('CharacterCount', () => {
   let $container
   let $textarea
+  let $textareaDescription
+
+  function cloneContainerAndAddToDocument() {
```

Comment:
Aware that we didn't need it previously, but with some tests continuing to do their own appending do we need a helper?

Reply:
It felt worth abstracting as all but one test use it straight away. The one that does not use it requires a parent wrapping the element. 

Now you mention it, though, wondering if injecting the HTML is not something that can be sorted in the global `beforeEach` that got introduce to reset the `document`. Each test could then do a `querySelector` similar to what we do in Puppeteer. Let me see if I can rework things in that direction and remove that helper :)